File: servers/dropwizard/src/main/java/com/slabstech/revive/server/dropwizard/retrofit/rx/GithubRxApp.java
package com.slabstech.revive.server.dropwizard.retrofit.rx;
import java.io.IOException;

public class GitHubRxApp {

    public static void main(String[] args) throws IOException {
        String userName = "eugenp";
        new GitHubRxService().getTopContributors(userName).subscribe(System.out::println);
    }
}

File: servers/dropwizard/src/main/java/com/slabstech/revive/server/dropwizard/retrofit/rx/GithubRxApp.kt
package com.slabstech.revive.server.dropwizard.retrofit.rx

import java.io.IOException

object GitHubRxApp {
    @Throws(IOException::class)
    @JvmStatic
    fun main(args: Array<String>) {
        val userName = "eugenp"
        GitHubRxService().getTopContributors(userName).subscribe(System.out::println)
    }
}